clippy_lints/src/methods/mod.rs

```diff
@@ -2931,7 +2933,7 @@ fn lint_find_map<'tcx>(
 ) {
     // lint if caller of `.filter().map()` is an Iterator
     if match_trait_method(cx, &map_args[0], &paths::ITERATOR) {
-        let msg = "called `find(p).map(q)` on an `Iterator`";
+        let msg = "called `find(..).map(..)` on an `Iterator`";
```

Comment:
Shouldn't it be
```suggestion
        let msg = "called `find(<p>).map(<q>)` on an `Iterator`";
```
to be consistent with other cases where there are more than one placeholder?

Reply:
Thanks, I missed this one

Reply:
Actually I did this because the suggestion is merging both `<p>`  and `<q>` with `.find_map(..)`, so it was weird to me that `p` and `q` disappeared like this in the suggestion.

Reply:
Oh I did not realize that. Although they were named before, I think you have a point :) Feel free to ignore my comments on similar cases then.